Editor/AvatarConfigEditor.cs

```diff
@@ -8,12 +9,29 @@ namespace ReadyPlayerMe.AvatarLoader.Core
     public class AvatarConfigEditor : Editor
     {
         private AvatarConfig avatarConfigTarget;
+        private SerializedProperty userDracoCompressionField;
```

Comment:
Do we need this SerializedProperty... or can we access the the property directly via the avatarConfigTarget assigned on line 16?

Reply:
Yep, with directly we don't get the value in field.

Reply:
I was wrong it does, but I had to check the value before OnInspectorGUI is called, which I do in OnEnable.